website/client/src/components/userMenu/profileModal.vue

```diff
@@ -55,6 +56,7 @@ export default {
     onHidden () {
       if (this.$route.path !== window.location.pathname) {
         this.$router.go(-1);
+        this.$root.$emit('restoreTitle');
```

Comment:
to follow internal conventions, can you rename the event `habitica: restoreTitle`?